src/redis_cmd.cc

```diff
@@ -143,21 +143,59 @@ class CommandKeys : public Commander {
 
 class CommandFlushDB : public Commander {
  public:
+  Status Parse(const std::vector<std::string> &args) override {
+     if (args.size() == 2 && !strcasecmp(args[1].data(), "async")) {
+       sync_ = false;
```

Comment:
It confused me that sync was determine whether to emit `compact` or not, instead of sync or async compact. Does it better to use other word like `with_compact` or others?